contents/blog/mixpanel-alternatives.md

```diff
@@ -0,0 +1,322 @@
+
+- 🔁 **Integration with Google tools** means it's easy to analyze your GA4 data elsewhere, such as Google's dashboarding tool, Looker Studio.
+
+- ⌨️ **Natural language search** means you can ask specific questions, like "MoM growth in users on iOS", rather searching existing reports – useful for answering quick, one-off queries.
+
+### Who uses GA4?
+
+Typical GA4 users are:
+- Marketing and content teams
+- Large news and content publishers
+- Enterprises who also use BigQuery
+
+Google's huge scale means GA4 is used by both the biggest global corporations to SMBs of every kind. It is ubiquitous.
+
+![GA4](../images/blog/mixpanel-alternatives/GA4.png)
+
+### Why do companies use GA4?
+
+1. **Because it's Google:** It's not an original reason, but it holds true. Using Google Analytics makes a lot of sense for teams who rely on other Google platforms, like Google Ads and BigQuery.
+
+2. **It's powerful and free:** Likewise, Google's scale means GA4 is completely free to most small and medium-sized businesses. This, combined with strong analytical tools and the large ecosystem of GA experts to call upon, makes it a safe choice.
+
+3. **To track marketing ROI:** While GA4 has pivoted toward product analytics, it's still predominantly used by marketing and e-commerce teams to track the ROI on marketing campaigns, and the performance of marketing websites. It's also popular among large content publishers for its scalability and content performance features.
+
+### How popular is GA4?
+
+Despite complaints from many users who preferred GA3, Google Analytics remains the most-used analytics tool in the world by a large margin. 
+
+As of July 2023, [30.5% of the top 1 million websites](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/) run GA4 – another 35.7% were still running GA3.
+
+### How does GA4 compare to Mixpanel?
+
+1. **Mixpanel is built for product teams:** Despite Google's overtures, GA4 still skews towards marketing and e-commerce use cases. Mixpanel is built for product teams and includes some useful collaboration features to support them.
+
+2. **GA4 is harder to use:** Mixpanel reviewers appreciate its easy-to-use UI. GA4, in contrast, has faced criticism for its complexity. Moving away from popular "pre-baked" reports in favor of more powerful, highly customizable ones has upset existing users.
+
+3. **Mixpanel is platform-agnostic:** While there's nothing stopping you from exporting GA4 data to non-Google platforms, it's mainly designed to play nicely with Google-owned products. Mixpanel happily integrates with a wide range or third-party tools across A/B testing, marketing automation, attribution, and beyond.
+
+> ### Bottom line
+>
+> GA4 is the best Mixpanel alternative for marketing teams who want to track ROI on campaigns.
```

Comment:
I'd add here how it's suited to teams who only want to track data in specific ways too

Reply:
What do you mean by "specific ways" here?

Reply:
As in, there are things you can't do in GA4 here. GA4 is the best Mixpanel alternative for marketing teams who want to track ROI data on Google ad campaigns, or simple web analytics metrics. It isn't suitable for those who want to explore deeper product metrics, or analyze data with SQL.